lib/Resource.js

```diff
@@ -167,34 +194,56 @@ function autoAssociate(resource) {
     return;
 
   _.forEach(resource.model.associations, function(association) {
-    // for prefetched data in list and read
-    if (!_.has(resource.associationOptions, 'autoPopulate') || !!resource.associationOptions.autoPopulate) {
-      if (!!association.as) {
-        resource.include.push({model: association.target, as: association.as});
-      } else {
-        resource.include.push(association.target);
+      // for prefetched data in list and read
```

Comment:
sorry to be such a stickler about this, but I do feel like it confuses code review a little bit to have these whitespace changes. Nothing actually changed in this section, but the whole section seems to have been needlessly shifted forward two spaces which is why its making it into the diff view here.

I just checked it locally, if you just de-indent everything within the `_.forEach(resource.model.associations, function(association) {` block and keep the `});` at the end like the original source, then this won't make it into the diff